earthquakes/python/publisher.py

```diff
@@ -1,15 +1,11 @@
 import json
 import asyncio
-import warnings
 from datetime import datetime, timedelta
 
 import requests
 from pyensign.events import Event
 from pyensign.ensign import Ensign
 
-# TODO Python>3.10 needs to ignore DeprecationWarning: There is no current event loop
-warnings.filterwarnings("ignore")
-
 # TODO: replace with YOU - your email and app details :)
 ME = "(https://rotational.io/data-playground/us-geological, earthquakes@rotational.io)"
```

Comment:
Should we add comments why this is being done ? why your email and app details are provided here ?

Reply:
Sure, I've added a comment.